scripts/west_commands/runners/core.py

```diff
@@ -649,6 +660,15 @@ def dev_id_help(cls) -> str:
                   target when multiple ones are available or
                   connected.'''
 
+    @classmethod
+    def extload_help(cls) -> str:
+        ''' Get the ArgParse help text for the --extload option.'''
+        return '''External loader to be used by stm32cubeprogrammer
+                  to program the targeted external memory.
+                  This must be given one time; the resulting argument
```

Comment:
I don't think we should specify `This must be given one time; `.
I guess this was useful for `tool-opt` which is a generic option, but in ext loader case it isn't useful.